apps/xmtp.chat/src/components/Messages/MessageContent.tsx

```diff
@@ -66,6 +71,15 @@ export const MessageContent: React.FC<MessageContentProps> = ({
     return <ReadReceiptContent />;
   }
 
+  if (contentType.sameAs(ContentTypeRemoteAttachment)) {
```

Comment:
Could we do something like `isRemoteAttachment(message)` here and then avoid the `content as RemoteAttachment` type casting?

Reply:
it actually doesn't work well here because the `DecodedMessage` is not available and passing in the values manually doesn't infer the correct type for `content`.